src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -1221,8 +1228,27 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 							} else {
 								result.add(key, ...folderTasks.set.tasks);
 							}
-							unUsedConfigurations.forEach((value) => {
+
+							const unUsedConfigurationsAsArray = Array.from(unUsedConfigurations);
+
+							const unUsedConfigurationPromises = unUsedConfigurationsAsArray.map(async (value) => {
 								let configuringTask = configurations!.byIdentifier[value];
+
+								for (const [handle, provider] of this._providers) {
+									if (configuringTask.type === this._providerTypes.get(handle)) {
+										try {
+											const resolvedTask = await provider.resolveTask(configuringTask);
+											if (resolvedTask) {
+												result.add(key, resolvedTask);
```

Comment:
This adds the task as a contributed task in the quick pick. We need to add it as a configuring task.